### DISCUSSION TITLE
Disable the Test/Prod option for the Production environment

### DISCUSSION THREAD

**Discussion by phpai** [openkfw/TruBudget#873] [General]
In GitLab by @jnogueira on Mar 14, 2019, 20:27

Currently, Test and Production environment are in the same server. Therefore, if you have a problem in this particular server, both enviroments can be lose.

This issue considers that there is a seggregation, one production server and other server for everything else at least.
In this scenario, it is important to disable the feature Test/Prod on the Production server. The Test/Prod combo still can be useful in the other environment (test, development, validation...), but no for Production.

Therefore, this issue requires to disable the Test/Prod feature for the Production environment.

![image (1)](https://user-images.githubusercontent.com/20185479/54906298-44c7d380-4ee3-11e9-9d68-5531686c7791.png)

**Comment by gonzochic** [comment]
From my standpoint I am strongly thinking about removing the selection of `prod` and `test` and simply deploying two separate frontends (one for `prod` and one for `test`). This would resolve this issue and also reduce confusion about which environment a user is connected to.

**Comment by josenogueiradalmeida** [comment]
The way we designed our environment in BNDES, we have: 
- one instance with Develop and User-Approval environments (Test and Prod combo options)
- one instance with Production environment (should have only Prod combo option)

Is there a way to make a parameter to enable those combo options? May be in the .env file?

Thanks!

**Comment by mathiashoeld** [comment]
@josenogueiradalmeida 
I think it would be possible to enable this via the .env file, but I think it would be better to be consistent and deploy separate frontends for each environment (see [comment above](https://github.com/openkfw/TruBudget/issues/133#issuecomment-486550179)) and completely remove the Prod/Test drop down menu.
